### ISSUE TITLE
Apache2 not start After upgrade or fresh install PHP 8.3 

### ISSUE THREAD

**Issue by omegacms** [termux/proot-distro#405]
First of all I want to say that I originally posted on the Termux bug track, but the guys there said that mine seems to be a proot-distro problem and advised me to write here.

### Problem description
I wanted to start porting my framework from php **8.2** to php **8.3**, but I had a problem with `termux/proot-distro`.

Practically termux/proot-distro is unable to manage the permissions needed to start apache2 after upgrading to PHP 8.3.

Nothing is recorded in the log files other than this warning located in the file `/var/log/apache2/error.log`

```php
Mon Apr 8 18:43:45 2024 (6346): Error Cannot create lock - Permission denied (13)
```

This error means that the `apache2.pid` file cannot be created in /var/run/apache2.

I would like to point out that up to PHP8.2 everything worked without problems.

I have tested:
Debian 12
Ubuntu 23.04
Ubuntu 23.10
OpenSUSE
Arch Linux
Fedora

With the same problem.

### What steps will reproduce the bug?

Step 1
Install a Linux distribution via proot-distro, manually or via any app present in F-Droid or Google Play.

Install apache2, change the port /etc/apache2/ports.conf from 80 to 8080 and start apache2 to test correct operation.

Step 2
Install php 8.3 from the packages of your distribution, stop and restarting Apache to see that after waiting 20 seconds Apache does not start and the message I wrote above appears in the /var/log/apache2/error.log file.

### What is the expected behavior?

I expect Apache and PHP to work fine.

**Comment by michalbednarski** [comment]
The issue should be fixed after `pkg up` to `proot 5.1.107-63`

**Comment by omegacms** [comment]
Works. Thank you all for your assistance and sorry for the inconvenience.
